docs/core/observability.md

```
@@ -0,0 +1,288 @@
+  code: 'UNAUTHORIZED',
+  message: 'User not authorized',
+  correlationId: ctx.correlationId,
+  meta: { userId: ctx.actor?.id },
+});
+```
+
+---
+
+## File Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)